userspace/libscap/engine/gvisor/CMakeLists.txt

```diff
@@ -47,3 +40,32 @@ add_custom_command(
     COMMAND ${PROTOC} -I ${CMAKE_CURRENT_SOURCE_DIR}/proto --cpp_out=. ${CMAKE_CURRENT_SOURCE_DIR}/proto/pkg/sentry/seccheck/points/syscall.proto
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
 )
+
+if (BUILD_SHARED_LIBS)
+    # Trying to build a shared scap_engine_savefile will result in circular
```

Comment:
Nit: comment talks about `scap_engine_savefile` instead of `gvisor`.

Reply:
Thanks for catching that. Fixed.